background.js

```diff
@@ -234,7 +234,8 @@ const files = [
     },
     {
         type: "css",
-        file: "./lib/quilljs/quill.snow.css"
+        file: "./lib/quilljs/quill.snow.css",
+        only: ["song", ...forumPages]
```

Comment:
improves loading speed + reduces memory overhead on non-song and non-forum pages